app/models/course/leaderboard_settings.rb

```diff
@@ -35,13 +35,48 @@ def display_user_count
     @settings.display_user_count || 30
   end
 
-  # Returns the number of users to be displayed on the leaderboard
+  # Set the number of users to be displayed on the leaderboard
   #
   # @param [Integer] count The number of users to be displayed
   def display_user_count=(count)
     @settings.display_user_count = count
   end
 
+  # Returns whether group leaderboard is enabled (disabled by default).
+  #
+  # @return [Boolean] option Setting on whether group leaderboard is enabled.
+  def display_group_leaderboard
+    @settings.display_group_leaderboard = false if @settings.display_group_leaderboard.nil?
```

Comment:
Might be better, because I was wondering how we can manage group leaderboard settings if the group leaderboard is disabled. I think this would make it easier for that.